ldes-fragmentisers/ldes-fragmentisers-geospatial/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/fragmentisers/geospatial/config/GeospatialProperties.java

```diff
@@ -6,6 +6,7 @@ private GeospatialProperties() {
 	}
 
 	public static final String PROJECTION = "projection";
+	public static final String NOOP = "noop";
```

Comment:
I would not place this under the GeospatialProperties as these are all keys. "noop" or "lambert72" are values for the key "projection"